modules-themes/readme.md

```diff
@@ -0,0 +1,42 @@
+# Listing third party open source and free of charge modules and themes for the PrestaShop e-commerce solution
```

Comment:
Another option
```suggestion
# List of third party, open source and free (as in "free beer") modules and themes for the PrestaShop e-commerce solution
```

Reply:
is "free of charge" not enough explicit?

Reply:
I don't know, it sounds weird to me